sdk/ai/ai-projects/samples-dev/agents/agents_azure_ai_search.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
The `agentCreateWithTracing` sample looks correct to me. The rest need to be renamed as follows:
- camelCase for files
- summary similar to https://github.com/Azure/azure-sdk-for-js/pull/32172/files#diff-8e091e55734cd6f476ee686da35028d5c6030f1ca5b80efeb2bdd0a46b1c381bR1-R9